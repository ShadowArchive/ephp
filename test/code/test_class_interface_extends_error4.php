<?php

interface A {
    public function abc($a);
}

interface B {
    public function abc($b, $c);
}

class C implements A, B {}

