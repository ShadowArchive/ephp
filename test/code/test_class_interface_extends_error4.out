
Fatal error: Declaration of A::abc() must be compatible with B::abc($b, $c) in {{CWD}}/test/code/test_class_interface_extends_error4.php on line 11
